### ISSUE TITLE
Inconsistent and incorrect test examples from `cartridge create`

### ISSUE THREAD

**Issue by t1-10** [tarantool/cartridge-cli#601]
`cartridge create` creates test case examples in `<app-name>/test/integration/api_test.lua` with Luatest hooks in this format (**format1**):

```lua
g.before_all = function() ... end
g.before_each = function() ... end
```

Both Luatest's README on GitHub and module reference on Tarantool docs site use this format (**format2**):

```lua
-- Hooks to run once for tests group
g.before_all(function() ... end)
g.after_all(function() ... end)

-- Hooks to run for each test in group
g.before_each(function() ... end)
g.after_each(function() ... end)
```

It looks like the format1 is wrong. A hook like `g.before_each = function() error("zzz") end` does not affect the tests at all. `g.before_all = function() error("xxx") end` does somehow work though 🤷‍♂️.

Please fix the templates for the example test cases. It would also be great to have them auto-tested in CI.

```console
$ cartridge --version
Tarantool Cartridge CLI v2.9.1 darwin/amd64 commit: 5770f5e
```

**Comment by mRrvz** [comment]
See https://github.com/tarantool/cartridge-cli/pull/557/files - we haven't just released it yet

**Comment by mRrvz** [comment]
Fixed at 2.10 - https://github.com/tarantool/cartridge-cli/releases/tag/2.10.0
